e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js

```diff
@@ -7,6 +7,7 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+// Stage: @prod
```

Comment:
T1672_1 from this spec has failed in all 5 latest runs. T1678 has been failing occasionally as well.

Reply:
Oh, right. I thought this https://github.com/mattermost/mattermost-webapp/pull/7765 was already merged.

Reply:
Now removed, PTAL, thanks Prapti!